src/index.ts

```diff
@@ -183,4 +183,18 @@ export class SilentPayment {
 
     return ret
   }
+
+  static isPaymentCodeValid(pc: string) {
+    try {
+      const result = bech32m.decode(pc, 118);
```

Comment:
IIRC, this 118 is the char limit being passed to decode? Since new versions of sp address can (and likely will) have more data added to the data part, might be better to use 1024 here? This is the recommended number from the BIP